### PR TITLE
Add new Fwlinks for v6 help content

### PR DESCRIPTION
**Pull request by adityapatwardhan**

### CODE REVIEW

**Comment by joeyaiello** [comment]
Didn't check them all, but I spot-checked a few and they look right to me. 

Are you planning on getting the content uploaded *before* the merge so that we know everything is passing? Or is there a chicken/egg problem there?

**Review by joeyaiello — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
Working on updating the help content with the correct fwlinks. We would need to move it to 'live' before these tests pass. Then we can merge.

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

File: src/Modules/Shared/PSReadLine/PSReadLine.psd1 [hunk 1]
```diff
@@ -13,5 +13,6 @@ CLRVersion = '4.0'
 FunctionsToExport = 'PSConsoleHostReadline'
 CmdletsToExport = 'Get-PSReadlineKeyHandler','Set-PSReadlineKeyHandler','Remove-PSReadlineKeyHandler',
                   'Get-PSReadlineOption','Set-PSReadlineOption'
-HelpInfoURI = 'https://go.microsoft.com/fwlink/?LinkId=528806'
+HelpInfoURI = 'https://go.microsoft.com/fwlink/?linkid=855966
+'
```
Comment (daxian-dbw) [inline src/Modules/Shared/PSReadLine/PSReadLine.psd1]: Is this quote put on a new line intentionally?

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
There are some failures in CI runs. Can you please fix them?

**Comment by adityapatwardhan** [comment]
The failures are because the help content is wrong. I am fixing it in a different repo.

File: src/Modules/Shared/PSReadLine/PSReadLine.psd1 (hunk 1 above)
Comment (adityapatwardhan) [reply src/Modules/Shared/PSReadLine/PSReadLine.psd1]: Fixed.

**Comment by adityapatwardhan** [comment]
@daxian-dbw I have updated PR and the required help content is also fixed. All tests pass now. Please have another look.

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**